src/System.Memory/dir.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.1.0</AssemblyVersion>
+    <AssemblyVersion>4.0.1.1</AssemblyVersion>
     <PackageVersion>4.5.1</PackageVersion>
```

Comment:
Do we know why there is a `PackageVersion` property both here, and in the .pkgproj?

Reply:
I saw that, but decided to leave it since the `pkgproj` overrides it. I am not sure why we have it in both places. It might have been an oversight when it was added to `dir.props` in https://github.com/dotnet/corefx/pull/29925. I don't think it's worth removing, but if you disagree, should remove it from the `pkgproj` and just keep the one in `dir.props` (or vice versa)?

We aren't consistent:
https://github.com/dotnet/corefx/blob/39fcb4b73c1a7dfca721d5f727603ffd35a428d8/src/System.Threading.Tasks.Extensions/pkg/System.Threading.Tasks.Extensions.pkgproj#L9
https://github.com/dotnet/corefx/blob/39fcb4b73c1a7dfca721d5f727603ffd35a428d8/src/System.Threading.Tasks.Extensions/dir.props#L6

cc @tarekgh 

See related thread where it was added to the `pkgproj`:
https://github.com/dotnet/corefx/pull/29970#discussion_r191842027